### PR TITLE
`docs example improvement`-` 30 file part1`

### PR DESCRIPTION
**Pull request by heiazuo**
public pr for `docs example improvement`-` 30 file part1`

### CODE REVIEW

File: website/docs/r/cosmosdb_account.html.markdown [hunk 1]
```diff
@@ -55,7 +55,7 @@ resource "azurerm_cosmosdb_account" "db" {
   }
 
   geo_location {
-    location          = var.failover_location
+    location          = azurerm_resource_group.example.location
```
Comment (ms-henglu) [inline website/docs/r/cosmosdb_account.html.markdown:58]: This should be another location.

File: website/docs/r/data_factory_custom_dataset.html.markdown [hunk 2]
```diff
@@ -42,18 +42,25 @@ resource "azurerm_data_factory_linked_custom_service" "example" {
   type                 = "AzureBlobStorage"
   type_properties_json = <<JSON
 {
-  "connectionString":"${azurerm_storage_account.test.primary_connection_string}"
+  "connectionString":"${azurerm_storage_account.example.primary_connection_string}"
 }
 JSON
 }
 
+
```
Comment (ms-henglu) [inline website/docs/r/data_factory_custom_dataset.html.markdown:50]: This new line should be removed.

File: website/docs/r/disk_encryption_set.html.markdown [hunk 3]
```diff
@@ -69,9 +69,15 @@ resource "azurerm_key_vault_access_policy" "example-disk" {
   object_id = azurerm_disk_encryption_set.example.identity.0.principal_id
 
   key_permissions = [
+    "Create",
+    "Delete",
     "Get",
-    "WrapKey",
-    "UnwrapKey"
+    "Purge",
```
Comment (ms-henglu) [inline website/docs/r/disk_encryption_set.html.markdown:75]: Does this resource requires these permissions?

**Review by ms-henglu — CHANGES_REQUESTED** [review]

**Review by katbyte — APPROVED** [review]
LGTM 💯

**Pull request merged**